metric-schema-api/src/main/conjure/metric-schema-api.yml

```diff
@@ -23,6 +23,7 @@ types:
         fields:
           type: MetricType
           tags: set<string>
+          values: map<string, set<TagValue>>
```

Comment:
Should this be a list/set e.g. `tagDefinitions: set<TagDefinition>` in order to support additional data on the tag itself (tag level documentation and future types of constraints, for example)?

That would allow us to deprecate and eventually remove the existing `tags` field. In the meantime we can continue to add all tag names to that set to avoid breaking compatibility.

Reply:
I've made the change as you've requested. I made it so that the Java generator ignores the old field (`tags`) since it should only ever be invoked with a  version of the compiler that populated `tagDefinitions`. The markdown generator supports both fields since it can be operating on old schemas that have not adopted the new structure

Reply:
:+1: